### DISCUSSION TITLE
I can't manage to show all the consoles

### DISCUSSION THREAD

**Discussion by Jbijjer** [EmulatorJS/EmulatorJS#782] [Q&A]
Hi!
I have upload roms on the NES, SNES and NDS emulators, but when I get to the main page, I can only see the NES emulator. Is there something I'm doing wrong?

Here's what is on my main config file :

{
  "title": "EmulatorJS",
  "root": "main",
  "parent": "main",
  "display_items": 3,
  "defaults": {
    "emulator": "",
    "bios": "",
    "path": "main",
    "rom_extension": "",
    "video_position": "",
    "type": "menu",
    "has_back": true,
    "has_corner": true,
    "has_logo": true,
    "has_video": true,
    "multi_disc": 0
  },
  "items": {
    "nds": {
      "video_position": "left:23.8vw;top:25.7vh;width:20vw;height:56vh;"
    },
    "nes": {
      "video_position": "left:11.5vw;top:31.5vh;width:36vw;height:43vh;"
    },
    "snes": {
      "video_position": "left:11.5vw;top:31.5vh;width:36vw;height:43vh;"
    }
  }
}

**Comment by ethanaobrien** [comment]
This doesn't look familiar.... is this linuxservers docker?
